frontend/main-app/src/components/Dashboard/Dashboard.tsx

```diff
@@ -24,12 +24,14 @@ export const Dashboard = (): React.JSX.Element => {
   }, []);
 
   return (
-    <main className={classes.dashboard}>
-      <DashboardOverview forms={forms} />
-      <div className={classes.formList}>
-        {forms.map((form) => (
-          <DashboardAccordion key={form.id} form={form} />
-        ))}
+    <main className={classes.dashboardContainer}>
```

Comment:
Same here regarding two mains :)
You can either remove the upper lvl main, or remove the lower lvl mains.
 
<img width="302" alt="image" src="https://github.com/ErlingHauan/FormFactory/assets/74791975/08e784c2-86a3-4a89-b221-dbe6d5c532fc">